sdk/js/util/http-mapper.js

```diff
@@ -29,7 +30,11 @@ function map (files) {
       result[service.name] = {}
 
       for (const method of service.methods) {
-        result[service.name][method.name] = { file: file.name, http: []}
+        result[service.name][method.name] = {
+          file: file.name,
+          http: [],
+          allowed_field_masks: allowedFieldMaskPaths[`/${service.fullName}/${method.name}`],
```

Comment:
I'd prefer lower camel case `allowedFieldMasks` here, which we use by default in the JS codebase (if not forced otherwise).